desktop/components/my_active_bids/helpers.coffee

```diff
@@ -1,6 +1,6 @@
 moment = require 'moment'
-{ PREDICTION_URL } = require('sharify').data
+Sale = require '../../models/sale.coffee'
 
 module.exports =
   liveAuctionUrl: (saleId) ->
-    "#{PREDICTION_URL}/#{saleId}"
+    (new Sale(id: saleId)).liveAuctionUrl(true)
```

Comment:
In this case, since it's trying to skip the `user` check and this was intentionally avoiding backbone models, I think it's fine to just return `"#{PREDICTION_URL}/#{saleId}/login"` here.